src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -47,7 +47,8 @@
 
     <div class="row">
       <div class="evaluee-col col-12">
-        <div class="row" *ngFor="let recipientSubmissionFormModel of model.recipientSubmissionForms; let i = index; trackBy: trackRecipientSubmissionFormByFn">
+        <div *ngFor="let recipientSubmissionFormModel of model.recipientSubmissionForms; let i = index; trackBy: trackRecipientSubmissionFormByFn">
+          <div class="row" *ngIf="formMode !== QuestionSubmissionFormMode.FLEXIBLE_RECIPIENT || i < model.customNumberOfEntitiesToGiveFeedbackTo">
```

Comment:
We do have a logic to generate `recipientSubmissionForms` and stop when it exceeds `customNumberOfEntitiesToGiveFeedbackTo`. I think the real bug might be there.